src/en/overlay/pimoroni-wide-input-shim.md

```diff
@@ -0,0 +1,38 @@
+<!--
+---
+name: Wide Input SHIM
+class: board
+type: power
+formfactor: Custom
+manufacturer: Pimoroni
+description: A power switch for the Raspberry Pi
```

Comment:
Is the description here correct, or was it copy-pasted from some other board?
Seems like this is actually a voltage-regulator board, rather than "just" a power switch?